Source/JavaScriptCore/wasm/js/WebAssemblyTablePrototype.cpp

```diff
@@ -95,13 +95,21 @@ JSC_DEFINE_HOST_FUNCTION(webAssemblyTableProtoFuncGrow, (JSGlobalObject* globalO
     RETURN_IF_EXCEPTION(throwScope, encodedJSValue());
 
     JSValue defaultValue = jsNull();
-    if (callFrame->argumentCount() < 2)
+    if (callFrame->argumentCount() < 2) {
+        if (Options::useWebAssemblyTypedFunctionReferences() && table->table()->wasmType().isRef())
```

Comment:
This would be clearer as just `!table->table()->wasmType().isNullable()` so I'll make that change later (also below).